openhands/server/session/session.py

```diff
@@ -197,3 +207,8 @@ def load_from_data(self, data: dict) -> bool:
             return False
         self.is_alive = data.get('is_alive', False)
         return True
+
+    def queue_status_message(self, message: str):
```

Comment:
We first might look at the granularity of messages we want to send, since I could imagine a couple more messages depending on the build status going on.
Right now, there are only like 3 status messages, which ain't much, but might suffice.

Reply:
Even if it's something like `starting_container`, `initializing_runtime`, etc that'd be perfect. Then we can move the english messages down into the FE where they can get into the translations

Reply:
Done: the sent messages are basically the actual translation keys now - loads of translations added for the FE :)